### ISSUE TITLE
JS API Documentation.

### ISSUE THREAD

**Issue by sustained** [carpiediem/memrise-enhancement-suite#3]
I've been trying to find some documentation for the Javascript API (such as the MEMRISE object). The MEMRISE object, especially, seems to have a lot of useful/interesting stuff from a quick glance such as functions for easily creating modals and so on and so forth.

Would anyone be interested in helping me try to decipher this? It would aid add-on developers greatly.

**Comment by carpiediem** [comment]
Have you seen [the wiki](https://github.com/carpiediem/memrise-enhancement-suite/wiki/Unofficial-Documentation-for-the-Memrise-API)?  It's quite old, but some of it may still be useful.  Feel free to make a PR if you see things that need changing.

**Comment by sustained** [comment]
Yeah, the wiki documents the REST API or the AJAX API or whatever you want to call it which is not at all what I'm talking about.

I'm talking about the frontend JavaScript API, not the backend API.
